masking_package/background_functions.py

```diff
@@ -0,0 +1,149 @@
+'''Background functions, functions the user does not have to know about'''
+
+def reverse_to_default(user_input, supposed_level, default_value):
+	'''Tests whether the user input is of the desired value level (e.g., bool).
+	If it is not, the default value of the package is used, instead.'''
+	if isinstance(user_input, supposed_level):
+		return user_input
+	else:
+		print(f'Warning! The value "{user_input}" is not a(n) {supposed_level}.'
+			  + f'The program used the default value "{default_value}", instead.')
+		return default_value
+
+def draw_stim(image_stim_name, default_image_stim, location):
```

Comment:
If I understand correctly, the user can choose their own image to be displayed, but can they also choose the location? If so, is it necessary to check whether their choice is appropriate, and also include a default value?